.github/workflows/golangci_lint.yml

```diff
@@ -36,4 +39,5 @@ jobs:
         uses: golangci/golangci-lint-action@v6
         with:
           version: latest
-          only-new-issues: true
+          # When is scheduled, show all issues
+          only-new-issues: ${{ github.event_name == 'schedule' && false || true }}
```

Comment:
If it works it works, but I don't quite understand how the logic works to toggle between true and false

Reply:
Yes, GitHub decided that this confusing syntax will be their ternary operator... So basically it means

```
false if github.event_name == 'schedule' else true 
```

Reply:
> ${{ github.event_name == 'schedule' && false || true }}

It's actually the equivalent syntax of:
```
${{ github.event_name == 'schedule' ? false : true }}
```
But the re-use of `||` and `&&`  (and also `true` and `false` in this case) make it particularly ugly.

Can you do this?
```
          only-new-issues: ${{ github.event_name != 'schedule' }}
```
Or will that only give you a boolean value but won't turn it into the string?

Reply:
Your suggestion works well @ggallen , so I'm adopting it.